README.md

```diff
@@ -11,23 +11,50 @@ npm install
 ./build-assets.sh
 ```
 
-Start PostgreSQL:
+Set up the env file
+
+```
+cp -p .env.example .env
+```
+
+Start PostgreSQL and the app:
 
 ```
 docker-compose up
 ```
 
+Set up database tables first time you set up the database
+
+```
+docker-compose run --rm web django-admin migrate
+docker-compose run --rm web django-admin createsuperuser
+```
+
+Load some development data
+
+```
+docker-compose run --rm web django-admin loaddata /root/.local/lib/python3.7/site-packages/khetha/fixtures/sample-task-data.json
```

Comment:
Note that `loaddata sample-task-data` suffices here: the file path is not needed.

(Django automatically discovers fixtures in each app's `fixtures` directory.)